app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
+    app.register_blueprint(descriptor_blueprint, url_prefix='/descriptor')
+
+    from resource import resource as resource_blueprint
+    app.register_blueprint(resource_blueprint, url_prefix='/resource')
+
+    from suggestion import suggestion as suggestion_blueprint
```

Comment:
Will the `suggestion` blueprint contain functionality both to publicly suggest new resources and to (as an admin) approve/deny suggested resources? Or just the latter? Personally, I think it makes sense to include both in this blueprint.